autobahn/wamp_session.ipp

```diff
@@ -550,6 +557,10 @@ boost::future<wamp_call_result> wamp_session<IStream, OStream>::call(
     msgpack::packer<msgpack::sbuffer> packer(*buffer);
     uint64_t request_id = ++m_request_id;
 
+    /*if (m_caller_disclose_me) {
+        options.set_disclose_me(true);
+    }*/
```

Comment:
Not picking any favorites, here are a few options I can think of right now:
* The easy way out: Making a copy of `options` and modifying that.
* The dynamic way: Adding a method like `default_call_options()` to `wamp_session` that the caller is expected to use. The options object returned would already include the disclosure preference, so it doesn't have to be added here.
* The intrusive way: Customizing the `wamp_call_options` pack algorithm, e.g. through a method `pack(Packer&, size_t num_extra_map_items, const std::function<void()>& pack_extra_map_items)` that's called instead of the regular `packer.pack(options)`.

I wonder how @oberstet feels about this.